test/test_other.py

```diff
@@ -15169,3 +15169,6 @@ def test_fp16(self, opts):
 
   def test_embool(self):
     self.do_other_test('test_embool.c')
+
+  def test_user_passed_lc(self):
+    self.run_process([EMCC, '-lc', '-sDISABLE_EXCEPTION_CATCHING=0', '-sMIN_SAFARI_VERSION=150000'])
```

Comment:
Aren't you missing a source file here?  What does this command produce.. I would expect to fail with "no input files".

Reply:
No, it passes:
`test_user_passed_lc (test_other.other) ... ok (1.02s)`
The program it outputs doesn't do anything though.

Reply:
Ah.. thats because we don't require a main function by default. 

We should probably include `test_file('hello_world.c')`.   In fact you might as well check that it runs too by using `do_runf` instead of `run_process`.

Reply:
The empty file does run, it exits with status 0. But sure, makes sense to add an actual program.